app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter!
+    author.update(posts_counter: author.posts.count)
+  end
+
+  def five_recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] Please use this method in your posts_index views for the best practices since you already created this before.